src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -962,6 +962,23 @@ public void ExtensionWarningEvent(string hubName, string functionName, string in
                 resource, attempt, delayMs, hubName, LocalAppName, LocalSlotName, ExtensionVersion);
         }
 
+        public void OrchestratorStateChange(
+            string hubName,
+            string functionName,
+            string instanceId,
+            FunctionType functionType,
+            FunctionState functionState,
+            bool isReplay)
+        {
+            if (this.ShouldLogEvent(isReplay))
+            {
+                this.logger.LogInformation(
+                    "{instanceId}: Function '{functionName} ({functionType})' state changed to {functionState}. IsReplay: {isReplay}. State: {state}. FunctionState: {functionState}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
+                    instanceId, functionName, functionType, functionState, isReplay, functionState, functionState, hubName,
+                    LocalAppName, LocalSlotName, ExtensionVersion, this.sequenceNumber++);
```

Comment:
It seems we're logging `functionState` twice, aren't we? Once for `State: ` and another time for `FunctionState: `.

Should we just log it in the `State: ` field?